src/osd/ECBackend.cc

```diff
@@ -1469,8 +1601,73 @@ void ECBackend::start_read_op(
   dout(10) << __func__ << ": started " << op << dendl;
 }
 
+void ECBackend::start_remaining_read_op(
+  ReadOp &op,
+  map<hobject_t, read_request_t, hobject_t::BitwiseComparator> &to_read)
+{
+  int priority = op.priority;
+  ceph_tid_t tid = op.tid;
+  op.to_read.swap(to_read);
+
+  dout(10) << __func__ << ": starting additional " << op << dendl;
+
+  map<pg_shard_t, ECSubRead> messages;
+  for (map<hobject_t, read_request_t>::iterator i = op.to_read.begin();
```

Comment:
i am a little bit surprised that this actually compiles. because `hobject_t` does not offer `operator<`, and neither do we define `std::less<hobject_t>` .

so `map<hobject_t, read_request_t>` should not compile without a comparator template parameter. even it is obvious to us that it's `hobject_t::BitwiseComparator` here.